submissions/TaniaBondarenko/popup2_task/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hooly-style Popup</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <!--Navigation panel with menu-button-->
+  <nav class="navigation">
+    <button class="nav_icon"><img src="images/photos.png" class="nav_icon_image"></button>
```

Comment:
Better to use list with links, not buttons, to make navigation, ex.:
``` 
    <nav>
        <ul>
            <li><a href="#"></a></li>
            <li><a href="#"></a></li>
        </ul>
    </nav>

Reply:
Thank you. Will be working on detailes.

Reply:
It works! Thank you!